src/test/java/com/google/sps/PlacesTest.java

```diff
@@ -89,8 +90,8 @@ public void getOnlyPlacesWithExpectedNumberOfRatingsAsHiddenGems() throws IOExce
   @Test 
   public void getRankedHiddenGemsBasedOnRatings() throws IOException {
     try (LocalTestServerContext sc = new LocalTestServerContext(AllPlacesApiNearbySearchRequest)) {
```

Comment:
this is optional but maybe sc could have a more meaningful name of the variable like serverContext

Reply:
done